### PR TITLE
docs(Ref): show again docs

### LINKED ISSUE #1774: Add documentation for `Ref` component to docsite
Add documentation for `Ref` component to docsite

### PR DESCRIPTION
**Pull request by layershifter**
Fixes #1774.

### CODE REVIEW

File: build/gulp/tasks/docs.ts [hunk 1]
```diff
@@ -79,7 +79,7 @@ task(
 
 const componentsSrc = [
   `${paths.posix.packageSrc('react')}/components/*/[A-Z]*.tsx`,
-  `${paths.posix.packageSrc('react-component-ref')}/src/[A-Z]*.tsx`,
+  `${paths.posix.packageSrc('react-component-ref')}/[A-Z]*.tsx`,
```
Comment (layershifter) [inline build/gulp/tasks/docs.ts:82]: Oops, broken path

File: build/gulp/plugins/util/getComponentInfo.ts [hunk 2]
```diff
@@ -74,10 +74,16 @@ const getComponentInfo = (filepath: string, ignoredParentInterfaces: string[]):
...
+  let type = componentType
   // add parent/child info
-  const isParent = filenameWithoutExt === dirname
+  let isParent = filenameWithoutExt === dirname
+
+  // Tweak for Ref component as it is distributed as a separate package
+  if (info.displayName === 'Ref') {
+    type = 'component'
+    isParent = true
+  }
```
Comment (layershifter) [inline build/gulp/plugins/util/getComponentInfo.ts:85]: Don't have better idea now

Comment (miroslavstastny) [reply build/gulp/plugins/util/getComponentInfo.ts:85]: 🐽

**Review by miroslavstastny — APPROVED** [review]
Changelog?

**Pull request merged**